MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -254,8 +255,14 @@ protected virtual Stream OpenStream(string assetName)
 			try
             {
                 string assetPath = Path.Combine(RootDirectory, assetName) + ".xnb";
-                stream = TitleContainer.OpenStream(assetPath);
 
+                // For Win8 metro applications you do not have access to Environment.CurrentDirectory nor does Path.DirectorySeparatorChar exist.
+#if !WINRT
+                if (Path.IsPathRooted(assetPath))
+                    assetPath = FileHelpers.GetRelativePath(Environment.CurrentDirectory + Path.DirectorySeparatorChar, assetPath);
```

Comment:
I agree, File.OpenRead ifdef'd for desktop platforms is a better approach. I'll make that change in a bit.